wappuapp-adminpanel/nginx/nginx.heroku.conf

```diff
@@ -0,0 +1,23 @@
+http {
+
+    upstream backend {
+        server ${BACKEND_URI};
+    }
+    
+    server {
+        listen ${PORT};
+        root /usr/share/nginx/html;
+
+        location /api/ {
+            proxy_pass http://backend/api/;
+            proxy_set_header Host http://${FRONTEND_URI};
+        }
+
+        location / {
+            try_files $uri /index.html;
+        }
+    }
+}
+
+
+events { }
```

Comment:
Github valittaa "No new line at end of file"

Reply:
korjataan